CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG
 
+## Next Version
```

Comment:
suggestion: Use "Unreleased" instead? I believe "Unreleased" is more common than "Next Version" but to be honest I'm fine with both 😄

Reply:
Sure.